base/arraymath.jl

```diff
@@ -24,7 +24,7 @@ julia> A
  2-2im  3-1im
 ```
 """
-conj!(A::AbstractArray{<:Number}) = (@inbounds broadcast!(conj, A, A); A)
+conj!(A::AbstractArray{<:Number}) = (@inbounds map!(conj, A, A); A)
```

Comment:
```suggestion
conj!(A::AbstractArray{<:Number}) = (map!(conj, A, A); A)
```

This caller doesn't know anything more than `map!` does (and I don't think `map!` uses a `@boundscheck` in any case).